azext_iot/common/certops.py

```diff
@@ -8,12 +8,18 @@
 certops: Functions for working with certificates.
 """
 
+import datetime
 from os.path import exists, join
 import base64
-from OpenSSL import crypto
```

Comment:
I believe this is our only import of `OpenSSL` - If so we can probably remove it from the `.pylintrc` ignored modules:
https://github.com/Azure/azure-iot-cli-extension/blob/58de288788d3d3bbe978f8311f47e9783aa0ca75/.pylintrc#L693